lib/web/integrations.go

```diff
@@ -504,3 +551,25 @@ func (h *Handler) integrationsMsTeamsAppZipGet(w http.ResponseWriter, r *http.Re
 	}
 	return nil, nil
 }
+
+func (h *Handler) integrationsExportCA(_ http.ResponseWriter, r *http.Request, p httprouter.Params, sctx *SessionContext, site reversetunnelclient.RemoteSite) (interface{}, error) {
```

Comment:
Is this necessary? Can we reuse our existing CA export endpoint?

Reply:
The current API gives a text file but the new API will be used for a guide enrolment flow in Web UI so it's better to return the original data for Web UI to decide how to display.